Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Expander/ExpanderXaml.bind

```diff
@@ -6,46 +6,46 @@
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     mc:Ignorable="d">
 
+  <Page.Resources>
+    <SolidColorBrush Color="{StaticResource SystemChromeLowColor}" x:Key="SystemControlForegroundChromeLowBrush"/>
+  </Page.Resources>
+
   <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
     <ScrollViewer>
       <StackPanel Margin="20">
         <controls:Expander x:Name="Expander1" VerticalAlignment="Top" Margin="0,0,0,10"
-                          Header="This is the header - expander 1"
-                          HorizontalContentAlignment="Stretch"
-                          Foreground="White"
-                          Background="@[BackgroundExpander1:Brush:Gray]"
-                          IsExpanded="@[IsExpanded1:Bool:False]@"
-                          ExpandDirection="@[ExpandDirection1:Enum:ExpandDirection.Down]">
+                           Header="This is the header - expander 1"
+                           HorizontalContentAlignment="Stretch"
+                           IsExpanded="@[IsExpanded1:Bool:False]@"
+                           ExpandDirection="@[ExpandDirection1:Enum:ExpandDirection.Down]">
           <Grid>
             <TextBlock HorizontalAlignment="Center"
-                      TextWrapping="Wrap"
-                      Text="This is the expanded content"
-                      VerticalAlignment="Center"
-                      Style="{StaticResource HeaderTextBlockStyle}" />
+                       TextWrapping="Wrap"
+                       Text="This is the expanded content"
+                       VerticalAlignment="Center" />
           </Grid>
 
           <controls:Expander.ContentOverlay>
-            <Grid Background="Red" MinHeight="250">
+            <Grid Height="120">
```

Comment:
We should stick with `MinHeight` here as otherwise it looks a bit strange if the expand direction is left or right.